File: trunk/src/test/java/net/esper/support/emit/SupportEmittedListener.java
package net.esper.support.emit;

import net.esper.client.EmittedListener;

import java.util.LinkedList;
import java.util.List;

public class SupportEmittedListener implements EmittedListener
{
    List<Object> emittedObjects = new LinkedList<Object>();

    public void emitted(Object object)
    {
        emittedObjects.add(object);
    }

    public List<Object> getEmittedObjects()
    {
        return emittedObjects;
    }
}

File: NEsper.Tests/support/emit/SupportEmittedListener.cs
using System;
using System.Collections.Generic;

using net.esper.client;

namespace net.esper.support.emit
{
    public class SupportEmittedListener
    {
        IList<Object> emittedObjects = new List<Object>();

        public virtual void HandleEmit(Object _object)
        {
            emittedObjects.Add(_object);
        }

        public IList<Object> getEmittedObjects()
        {
            return emittedObjects;
        }
    }
}
